base/docs/helpdb/Base.jl

```diff
@@ -786,6 +859,48 @@ at the position where it would appear if the array were fully sorted via a non-s
 algorithm. If `k` is a single index, that value is returned; if `k` is a range, an array of
 values at those indices is returned. Note that `select!` does not fully sort the input
 array.
+
+# Examples
+
```

Comment:
Extra empty line here as well? (Have we a convention for absence/presence of an extra empty line following `# Example[s]`? Remarked here and above for consistency with the absence earlier in this PR, but have no preference.)